### PR TITLE
Fixed outdated rpc-svcgssd service name (bsc#972488)

### PR DESCRIPTION
**Pull request by cwh42**

### CODE REVIEW

File: src/modules/NfsServer.rb [hunk 1]
```diff
@@ -277,28 +278,28 @@ def Write
...
-          if !Service.active?("svcgssd")
-            unless Service.Start("svcgssd")
-              # FIXME svcgssd is gone! (only nfsserver is left)
+          if !Service.active?(GSSERVICE)
+            unless Service.Start(GSSERVICE)
+              # FIXME #{GSSERVICE} is gone! (only nfsserver is left)
               Report.Error(
                 _(
-                  "Unable to start svcgssd. Ensure your kerberos and gssapi (nfs-utils) setup is correct."
+                  "Unable to start #{GSSERVICE}. Ensure your kerberos and gssapi (nfs-utils) setup is correct."
```
Comment (dgdavid) [inline src/modules/NfsServer.rb]: We cannot use interpolation in translations.

See [Travis complain](https://travis-ci.org/yast/yast-nfs-server/builds/637522222#L166)

> Note: #{foo} substitution in translatable strings does not work properly, use
> 
>  _("foo %{bar} baz") % { :bar => bar }
>
> or
> 
>  _("foo %s baz") % bar

and also the explanation [at YaST Documentation site](https://yastgithubio.readthedocs.io/en/latest/localization/#interpolations).

File: src/modules/NfsServer.rb [hunk 2]
```diff
@@ -277,28 +278,28 @@ def Write
...
                 )
               )
               ok = false
             end
           else
-            unless Service.Restart("svcgssd")
+            unless Service.Restart(GSSERVICE)
               Report.Error(
-                _("Unable to restart 'svcgssd' service.")
+                _("Unable to restart '#{GSSERVICE}' service.")
```
Comment (dgdavid) [inline src/modules/NfsServer.rb]: The same localization/interpolation problem. See above.

File: src/modules/NfsServer.rb [hunk 3]
```diff
@@ -277,28 +278,28 @@ def Write
...
               ok = false
             end
           end
         else
-          if Service.active?("svcgssd")
-            unless Service.Stop("svcgssd")
-              Report.Error(_("'svcgssd' is running. Unable to stop it."))
+          if Service.active?(GSSERVICE)
+            unless Service.Stop(GSSERVICE)
+              Report.Error(_("'#{GSSERVICE}' is running. Unable to stop it."))
```
Comment (dgdavid) [inline src/modules/NfsServer.rb]: The same localization/interpolation problem. See above.

**Comment by dgdavid** [comment]
BTW, maybe we can fix https://github.com/yast/yast-nfs-server/pull/39 first and then make forward ports according to the [maintenance documentation](https://yastgithubio.readthedocs.io/en/latest/maintenance-branches/).

What do you think?

**Comment by ancorgs** [comment]
I will take over to make all the suggested changes

**Comment by ancorgs** [comment]
Closed in favor of #41, that includes all these commits and also fixes the problems found by Travis.

**Pull request closed without merging**